src/main/scala/net/ripe/rpki/publicationserver/repository/DataFlusher.scala

```diff
@@ -24,73 +24,54 @@ class DataFlusher(conf: AppConfig)(implicit val system: ActorSystem,  val health
   implicit val executionContext = system.dispatcher
 
   protected lazy val rrdpWriter = new RrdpRepositoryWriter
-  protected lazy val rsyncWriter = new RsyncRepositoryWriter(conf)
 
   lazy val pgStore = PgStore.get(conf.pgConfig)
 
   // The latest serial for which version freeze was initiated by this server instance.
   private var latestFrozenSerial : Option[Long] = None
 
-  // Write initial state of the database into RRDP and rsync repositories
+  // Write initial state of the database into the RRDP repository
   // Do not change anything in the DB here.
   def initFS() = try {
-    // do not do anything at all if neither `writeRsync` nor `writeRrdp` is set.
-    if (conf.writeRsync || conf.writeRrdp) {
```

Comment:
I would keep `conf.writeRrdp` for now.
It may be useful if we ever want to have configuration where more than one server work with the same database and one of them generates RRDP files and others only accept publish requests. That was the intention of having these flags.